### DISCUSSION TITLE
html-tag and body-tag in $mail->body

### DISCUSSION THREAD

**Discussion by fuehlersysteme** [PHPMailer/PHPMailer#3051] [Q&A]
Hello,

is it possible to pass the html and body tag as html to $mail->body ? 
Or is this then included twice?

**Comment by Synchro** [comment]
`$mail->Body` refers to the email message body; it has nothing to do with the HTML tag of the same name. You can of course put HTML into `Body`, for example:

```php
$mail->Body = '<html><body>Hello</body></html>';
